src/types/api.ts

```diff
@@ -318,3 +318,34 @@ export type MediaGridContract = {
   address: string;
   chain: string;
 };
+
+export type TransactionHistory = {
+  results: TransactionHistoryItem;
+};
+
+export type TransactionHistoryItem = {
+  outgoing: Transaction[];
+  incoming: Transaction[];
+};
+
+export type Transaction = {
```

Comment:
Can you rename this to `FlairTransaction` please? The reason for this is because this data type is specific to Flair and we will have different shapes of transactions from Etherscan, for example. In that scenario there would be an additional type called EtherscanTransaction and it would be easier to differentiate between the two 🙏

Reply:
Done